docs/guides/mozilla-sops.md

```diff
@@ -104,6 +88,81 @@ flux create kustomization my-secrets \
 Note that the `sops-gpg` can contain more than one key, sops will try to decrypt the
 secrets by iterating over all the private keys until it finds one that works.
 
+## Optional: Export the Public Key into the git directory
```

Comment:
```suggestion
## Optional: Export the Public Key into the Git directory
```

Git is used as a name here, `git` is the command. Also not sure why "Public Key" is used in the title, but throughout the text it is styled as "public key".